Added setup of external route for Defenders comms

azaceta opened the pull request:
I found the current instructions for OpenShift 4 didn't allow the communication of external Defenders back to the Console, so I:
Added the definition of an external route to expose port 8084 to enable communication of Defenders to the Console.
Added setup of the route's URL in the SAN of the console, and in the Defender installation instructions, also adding the option to specify the target port from the external Defender point of view.

azaceta committed:
Added setup of external route for Defenders comms

Added the definition of an external route to expose port 8084 to enable communication of Defenders to the Console.
Added setup of the route's URL in the SAN of the console, and in the Defender installation instructions, also adding the option to specify the target port from the external Defender point of view.

wfg committed:
Make tiny edits

wfg reviewed (approved):
@azaceta Thanks for the addition. LGTM!

@pfox1969 one more set of eyes please?

pfox1969 committed:
Clarifications on exposing the external Defender route

Updated with a little clarification of 
1) the Defender <-> Console external route must be TLS passthrough
2) need to update the daemonSet yaml's wss path to the external router's 443 TCP port 
We used to publish these steps on TechDocs https://docs.paloaltonetworks.com/prisma/prisma-cloud/19-11/prisma-cloud-compute-edition-admin/howto/openshift_external_defenders

wfg commented:
@Pubs-MV good to go 👍
